wrapper/Sarek.egg-info/PKG-INFO

```diff
@@ -0,0 +1,45 @@
+Metadata-Version: 1.0
+Name: Sarek
+Version: 0.1.dev0
+Summary: Helper tools for use with the Sarek Nextflow pipeline.
+Home-page: https://github.com/SciLifeLab/Sarek
+Author: Maxime Garcia, Szilvester Juhos, Phil Ewels
+Author-email: maxime.garcia@scilifelab.se, phil.ewels@scilifelab.se
+License: MIT License
+
+Copyright (c) 2016 SciLifeLab
```

Comment:
It’s 2018 :-)

Reply:
Oops, those files shouldn't even be in there..!

Reply:
Ok, licence file also updated to 2018 ;)

![party](https://i.pinimg.com/236x/5d/c1/47/5dc147580ab7ebf0c58d9d049765d64b.jpg)